service_taxi/src/main/kotlin/org/thechance/service_taxi/data/DataBaseContainer.kt

```diff
@@ -0,0 +1,9 @@
+package org.thechance.service_taxi.data
+
+import org.koin.core.annotation.Single
+import org.litote.kmongo.coroutine.CoroutineClient
+
+@Single
+class DataBaseContainer(client: CoroutineClient) {
```

Comment:
For consistency in naming, it would be better renaming CoroutineClient to MongoClient.